frontend/src/components/common/form/styles.ts

```diff
@@ -60,3 +60,11 @@ export const FormSection = styled.div`
   padding: 2.5rem;
   background-color: ${props => props.theme.css.filterBackgroundColor};
 `;
+
+export const FormSectionClear = styled.div`
```

Comment:
nit: it took me a while to get why this was named "__Clear". If I understood correctly it wanted to indicate that this is a FormSection with no background. Would you consider a name closer to "FormSectionNoBg" ?

Reply:
That's right, just needed one without the background. I don't have a strong opinion either way.